assets/lang/sklmessages_pl_PL.properties

```diff
@@ -83,21 +83,21 @@ settings.title=Ustawienia launchera
 settings.lang=Język
 settings.lang.help=Widzisz błąd w tłumaczeniu?
 settings.lang.link=Pomóż go naprawić!
-settings.theme=Motyw
+settings.theme=Motywy
 settings.theme.light=Jasny motyw
 settings.theme.dark=Ciemny motyw
 settings.token=Specjalny klucz dostępu
 settings.alert.title=Informacja
-settings.alert.text=Musisz zrestartować launcher, aby zastosować zmiany.
+settings.alert.text=Musisz uruchomić ponownie launcher, aby zastosować zmiany.
```

Comment:
Nie jestem przekonany co do tej zmiany. Nie ma tutaj niczego złego, lecz "zrestartować" brzmi jakoś tak bardziej profesjonalnie (?). Ale to kwestia gustu. @Waterrail co o tym myślisz?

Reply:
Moim zdaniem 'zrestartować' brzmi lepiej. W końcu każdy wie, co to znaczy, a jest krócej